### PR TITLE
Bug 1535976 - Fix input name in the create-secret form

### PR DESCRIPTION
**Pull request by jhadvig**
Fixed wrong `input` names in the `create-secret.html` 
@jwforres PTAL

### CODE REVIEW

**Comment by jwforres** [comment]
the bug says webhook secret key is not being validated, I dont see how this fixes that issue?

**Comment by jhadvig** [comment]
@jwforres so since we are not doing any kind of validation of the secret key, but I've noticed that the secret name was not validated properly, since the input name was different from the name that that was checked on the form (and for some other input fields as well).
Sorry I've forgot to clarify it in the first comment.

**Comment by jwforres** [comment]
ok i know on the original set of changes i said not to worry about the validation, but now we can probably look into it as a follow-on here. webhook secret key needs to be a valid URL path segment. Thats probably the only validation we should try to do.

**Comment by jhadvig** [comment]
@jwforres PTAL

File: app/scripts/directives/createSecret.js [hunk 1]
```diff
@@ -26,6 +26,11 @@ angular.module("openshiftConsole")
       templateUrl: 'views/directives/create-secret.html',
       link: function($scope) {
         $scope.nameValidation = DNS1123_SUBDOMAIN_VALIDATION;
+        $scope.secretReferenceValidation = {
+          pattern: /^[a-zA-Z0-9\-_]+$/,
+          minLength: 8,
+          description: 'Secret reference key must consist of lower-case, upper-case letters, numbers, bash and underscore.'
```
Comment (jwforres) [inline app/scripts/directives/createSecret.js]: dash not bash, also we use oxford commas here

"dash, and underscore"

File: app/views/directives/create-secret.html [hunk 2]
```diff
@@ -70,6 +72,16 @@
...
           <a ng-href="{{'webhooks' | helpLink}}" target="_blank"><span class="learn-more-inline">Learn More&nbsp;<i class="fa fa-external-link" aria-hidden="true"></i></span></a>
         </div>
+        <div class="has-error" ng-show="secretForm.webhookSecretKey.$error.pattern && secretForm.webhookSecretKey.$touched">
+          <span class="help-block">
+            {{secretReferenceValidation.description}}
+          </span>
+        </div>
+        <div class="has-error" ng-show="secretForm.webhookSecretKey.$error.minlength && secretForm.webhookSecretKey.$touched">
+          <span class="help-block">
+            Minimum character count is {{secretReferenceValidation.minLength}}.
```
Comment (jwforres) [inline app/views/directives/create-secret.html]: looks like you got this message from the env editor, but I prefer what we say in other places which is something like:

`The key must have at least 8 characters.`

File: app/scripts/directives/createSecret.js (hunk 1 above)
Comment (jhadvig) [reply app/scripts/directives/createSecret.js]: always forgetting about the commas. We dont use int with 'and' ... sorry

**Comment by jhadvig** [comment]
@jwforres comments addressed. added dist as well.

**Pull request merged**